src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java

```diff
@@ -0,0 +1,23 @@
+package de.mediathekview.mserver.crawler.sender.hr;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "HrSendungenDto.java" should be located in "de/mediathekview/mserver/crawler/sender/hr" directory, not in "/home/travis/build/mediathekview/MServer/src/main/java/mServer/crawler/sender/hr". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)